apps/firecrawler/.env.example

```
@@ -0,0 +1,292 @@
+# - Verify OPENAI_API_KEY or OLLAMA_BASE_URL configuration
+# - Check if your Firecrawl instance has vector data
+# - Try lowering FIRECRAWLER_VECTOR_SIMILARITY_THRESHOLD
+
+# SECURITY BEST PRACTICES:
+# ✓ Never commit .env files to version control
+# ✓ Use strong, unique tokens for FIRECRAWLER_AUTH_TOKEN
+# ✓ Regularly rotate API keys and tokens
+# ✓ Restrict FIRECRAWLER_ALLOWED_IPS in production
+# ✓ Use HTTPS in production (configure reverse proxy)
+# ✓ Monitor access logs for suspicious activity
+# ✓ Keep dependencies updated for security patches
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)